packages/design-system-react-native/package.json

```diff
@@ -47,6 +47,7 @@
     "test:watch": "NODE_OPTIONS=--experimental-vm-modules jest --watch"
   },
   "dependencies": {
+    "@metamask/design-system-shared": "workspace:^",
```

Comment:
Adding `@metamask/design-system-shared` which is now the source of truth for CAIP utils that can be shared across platforms